unitTests/ui/OmniboxTest.ts

```diff
@@ -28,6 +31,21 @@ export function OmniboxTest() {
       Simulate.removeJQuery();
     });
 
+    function setupEnv() {
+      testEnv = new Mock.MockEnvironmentBuilder();
+      testEnv.searchInterface.getOmniboxAnalytics = jasmine.createSpy('omniboxAnalytics').and.returnValue(new OmniboxAnalytics()) as any;
```

Comment:
Hmm, I am not sure about this line. It is good to mock, but here it seems we are returning an instance of the real class? I am not clear on why?

Reply:
Short answer, how do I Mock it 😆 

`OmniboxAnalytics` is not a component. It is just a simple class.
I didn't see anything in `Mock` to mock it.

Also, I did it the same way I did for the `MissingTermManager` which I also didn't mock